### PR TITLE
[ENH] Convert SlidingWindowTransformer to use numpy internally

### PR DESCRIPTION
**Pull request by TonyBagnall**
simple change, since the first line in fit was to convert to numpy!

#### Reference Issues/PRs
#197

### CODE REVIEW

File: aeon/classification/distance_based/_shape_dtw.py [hunk 1]
```diff
@@ -253,9 +252,10 @@ def _preprocess(self, X):
         # the test/training data. It extracts the subsequences
         # and then performs the shape descriptor function on
         # each subsequence.
-        X = convert(X, from_type="numpy3D", to_type="nested_univ")
+        # X = convert(X, from_type="numpy3D", to_type="nested_univ")
```
Comment (GuzalBulatova) [inline aeon/classification/distance_based/_shape_dtw.py:255]: Is this line commented out on purpose or can be deleted?

File: setup.cfg [hunk 2]
```diff
@@ -12,14 +12,6 @@ addopts =
     --ignore examples
     --ignore docs
     --doctest-modules
-    --durations 10
```
Comment (GuzalBulatova) [inline setup.cfg:15]: Just curious, what is the reason for deleting this pytest options?

**Review by GuzalBulatova — APPROVED** [review]
Looks good to go! Only have a couple of questions, not blockers.

File: setup.cfg (hunk 2 above)
Comment (TonyBagnall) [reply setup.cfg:15]: there is no reason, its a mistake I keep forgetting to correct, thanks!

File: aeon/classification/distance_based/_shape_dtw.py (hunk 1 above)
Comment (TonyBagnall) [reply aeon/classification/distance_based/_shape_dtw.py:255]: it can go, shapeDTW needs to be done after all these, thank you.

**Review by chrisholder — APPROVED** [review]

**Pull request merged**